source/extensions/filters/network/redis_proxy/BUILD

```diff
@@ -8,12 +8,33 @@ load(
 
```

Comment:
Optional, but I'd love to encourage commenting what code lives in what directories in top level build files.  Maybe a link to docs since most filters should already have them?

\# This is where redis functionality goes.  See https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/redis